index.html

```diff
@@ -41,5 +41,163 @@ <h2>I'am a Software Developer</h2>
+        <h6>work articles</h6>
+    
+        <article>
+            <h3>Profesional Art Printing Data</h3>
+
+            <p>A daily selection of privately personalized reads; no 
+                accounts or sign-ups required. has been the industry's
+                 standard</p>
+
+                 <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul> 
+                  <a href="#">See project</a>
+        </article>
+        <article>
+            <h3>Profesional Art Printing Data</h3>
+
+            <p>A daily selection of privately personalized reads; no 
+                accounts or sign-ups required. has been the industry's
+                 standard</p>
+
+                 <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul> 
+                  <a href="#">See project</a>
+        </article>
+        <article>
+            <h3>Profesional Art Printing Data</h3>
+
+            <p>A daily selection of privately personalized reads; no 
+                accounts or sign-ups required. has been the industry's
+                 standard</p>
+
+                 <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul> 
+                  <a href="#">See project</a>
+        </article>
+        <article>
+            <h3>Profesional Art Printing Data</h3>
+
+            <p>A daily selection of privately personalized reads; no 
+                accounts or sign-ups required. has been the industry's
+                 standard</p>
+
+                 <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul> 
+                  <a href="#">See project</a>
+        </article>
+        <article>
+            <h3>Profesional Art Printing Data</h3>
+
+            <p>A daily selection of privately personalized reads; no 
+                accounts or sign-ups required. has been the industry's
+                 standard</p>
+
+                 <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul> 
+                  <a href="#">See project</a>
+        </article>
+        <article>
+            <h3>Profesional Art Printing Data</h3>
+
+            <p>A daily selection of privately personalized reads; no 
+                accounts or sign-ups required. has been the industry's
+                 standard</p>
+
+                 <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul> 
+
+                  <a href="#">See project</a>
+        </article>
+
+    </section>
+
+    <section class="about-container">
+        <h6>skills</h6>
+        <article>
```

Comment:
Kindly use `div` tag as described before 😀

Reply:
Thanks very much for the review. I like the way you did it.